src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/utils/VisitBalancesUtil.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.utils
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.prison.api.VisitBalancesDto
+import java.time.LocalDate
+
+@Component
+class VisitBalancesUtil {
+  fun calculateAvailableVos(visitBalance: VisitBalancesDto?): Int {
+    return (visitBalance?.remainingVo ?: 0) + (visitBalance?.remainingPvo ?: 0)
+  }
+
+  fun calculateVoRenewalDate(visitBalance: VisitBalancesDto?, dateFrom: LocalDate = LocalDate.now()): LocalDate {
+    val latestIepAdjustDate = calculateIepAdjustDate(visitBalance, dateFrom)
+    val latestPrivilegedIepAdjustDate = calculatePrivilegedIepAdjustDate(visitBalance, dateFrom)
+
+    return if (latestIepAdjustDate.isBefore(latestPrivilegedIepAdjustDate)) {
```

Comment:
Could be simplified to 

return minOf(latestIepAdjustDate, latestPrivilegedIepAdjustDate)

which returns the soonest date.

Reply:
thanks, that's much neater